t/e2e.t

```diff
@@ -443,7 +443,7 @@ subtest "slow-start" => sub {
 
     my $each_cc = sub {
         my $cb = shift;
-        for my $cc (qw(reno pico cubic)) {
+        for my $cc (qw(reno cubic)) {
```

Comment:
@parvit quick question, is the change from master branch ? or our changes ? this test was introduced by other people. The test case we added is the "slow start search"

Reply:
You're absolutely right.